test/utils/MaterializerSupport.scala

```diff
@@ -17,9 +17,7 @@
 package utils
 
 import akka.actor.ActorSystem
-import akka.stream.ActorMaterializer
 
 trait MaterializerSupport {
```

Comment:
I would question the need for this trait at all. Not sure what its purpose is. Could you try removing this, and usages of it.